sdk/network/Azure.ResourceManager.Network/src/Generated/Models/NetworkIntentPolicyConfiguration.cs

```diff
@@ -18,6 +18,11 @@ public NetworkIntentPolicyConfiguration()
         /// <summary> The name of the Network Intent Policy for storing in target subscription. </summary>
         public string NetworkIntentPolicyName { get; set; }
         /// <summary> Source network intent policy. </summary>
-        public NetworkIntentPolicy SourceNetworkIntentPolicy { get; set; }
+        internal NetworkIntentPolicy SourceNetworkIntentPolicy { get; set; }
+        /// <summary> A unique read-only string that changes whenever the resource is updated. </summary>
+        public string SourceNetworkIntentEtag
```

Comment:
probably `SourceNetworkIntentPolicyEtag` is better? since it's `Etag` of `SourceNetworkIntentPolicy`, not `Etag` of `SourceNetworkIntent`.

Reply:
Good catch this was a bug where the initial property had a single property but I wasn't checking inheritance.